Modals/Fedosieiev Yehor - modals_task2.md

```diff
@@ -0,0 +1,51 @@
+#### Exercise 1. Choose the correct word or phrase. If both options are correct explain why.
+
+1. Hello. **Could** I speak to Mrs. Johnson, please?
+2. We **were allowed to** go home early yesterday because our techer was ill.
+3. The head teacher said we **could/were allowed to** go home.
+4. Do you think I **should** be worried about these spots on my forehead.
+5. You **ought to/should** enter that talent contest.
+6. You **shouldn't** talk to people like that. It's rude!
+7. Alan should **have written** two essays in the exam yesterday, not one.
+8. No, you **may** not go out tonight. You know you are grounded.
+9. Diana should have **waited/been waiting** for me at the corner. I wonder where she went.
+10. Yes, of course you **can** open the window if you're too hot!
+
+#### Exercise 2. Write a form of _must/have to/need to_ in each gap to complete the sentence.
+
+1. Oh, I must remember to get some potatoes on the way home tonight.
+2. Jason needs/has to see the headmaster during the next break. I wonder what's it about?
```

Comment:
has to - is the correct one as there is no necessity but a kind of obligation to visit the principal